server/src/main/java/org/opensearch/rest/action/admin/cluster/RestNodesStatsAction.java

```diff
@@ -185,6 +195,19 @@ public RestChannelConsumer prepareRequest(final RestRequest request, final NodeC
                     )
                 );
             }
+            if (metrics.contains("rest_actions") && request.hasParam("rest_actions_filters")) {
+                nodesStatsRequest.addRestActionsFilters(request.paramAsStringArray("rest_actions_filters", null));
+            } else if (request.hasParam("rest_actions_filters")) {
```

Comment:
Do we need validations on the provided values for `rest_actions_filters` as well?

Reply:
I did have thoughts about it while coding but the thing is we need to maintain the list of all rest actions and whenever a new rest action would be introduced, it must be added to that list. So I think it's un-necessary. As per current implementation if filter contain invalid action name it won't be returned as response.

Or we have to use RestActionsService (class in which all handlers are being registered) ref here, again it creates un-necessary dependency which I think is not a good design.